processor/groupbytraceprocessor/config.go

```diff
@@ -0,0 +1,46 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package groupbytraceprocessor
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config is the configuration for the processor.
+type Config struct {
+	configmodels.ProcessorSettings `mapstructure:",squash"`
+
+	// NumTraces is the max number of traces to keep in memory waiting for the duration.
+	// Default: 1.000.000.
+	NumTraces int `mapstructure:"wait_duration"`
```

Comment:
Looks like mapstructure tag is wrong?

Reply:
To be honest, it's the first time I've seen `mapstructure` and wasn't able to quickly find documentation about it. It did seem a bit counter-intuitive to me at first, and as I checked other processors, I noticed that they use the same tag for literal types as well:

https://github.com/open-telemetry/opentelemetry-collector/blob/8693118652c9ccabef31ffc76320251cb0379fcf/processor/samplingprocessor/tailsamplingprocessor/config.go#L80-L95

https://github.com/open-telemetry/opentelemetry-collector/blob/8693118652c9ccabef31ffc76320251cb0379fcf/processor/batchprocessor/config.go#L23-L36

https://github.com/open-telemetry/opentelemetry-collector/blob/8693118652c9ccabef31ffc76320251cb0379fcf/processor/queuedprocessor/config.go#L23-L35

Reply:
I meant it's mapping to wait_duration instead of num_traces.

Reply:
ouch, looks like I missed this twice :-) PR updated.